### DISCUSSION TITLE
Clicking Color on One Skin Sets It in All Skins

### DISCUSSION THREAD

**Discussion by ceylina** [SFX-WoW/Masque#7] [Questions] [answered]
I had to try this several times to make sure it was really happening but I can reproduce it now with almost certainty.

If you have a skin installed (may not be a criteria but it  is what I have), in my case your skin Caith, this happens when you even click ok after opening the color picker...

Note: I  tried several other skins as well to make sure, Caith is  not a factor in this case, even having no skins causes this.

Now this also may be the intended use but as skins have different intent, it doesn't seem right that this is the case...

Here  is the issue:

Open  Masque settings

without choosing a skin, go to colors and click normal, click ok on the now opened color picker

Choose another skin that would normally have another color in normal, it is now that first skin's color

You do not have to change the normal color, simply lick ok.

Back to Caith for example, it's normal color is a grey but if you say go to "blizzard", open the normal color picker, click ok then come back to Caith, that normal is now white.

**Comment by ceylina** [comment]
However, if you are within a skin and change the normal color (or other color) for a specific element of an addon that is masque enabled (let's say dominos and I change the normal color for action bars to a green, the normal color on other sections does not also turn green)

So it is just between skins and not within the addons that are enabled for masque within that skin.

**Comment by StormFX** [comment, marked as answer]
Actually, that _is_ the intended behavior. Colors are saved at the profile level, not the skin level. Saving the settings per skin would cause way too much bloat when you can simply reset the color to the skin's default when you select a new skin.

**Comment by ceylina** [comment]
Ah good to know. I'm self closing this issue then ;)
